tools/joystick/joystickd.py

```diff
@@ -16,42 +16,39 @@ def __init__(self, use_keyboard=True):
     self.use_keyboard = use_keyboard
     if self.use_keyboard:
       self.kb = KBHit()
-      self.buttons = {'c': 'cancel', 'e': 'engaged_toggle', 'r': 'reset'}
-      axes_buttons = {'gb': ['w', 's'], 'steer': ['a', 'd']}
-      self.axes = {key: ax for ax in axes_buttons for key in axes_buttons[ax]}
+      self.buttons = {'c': 'cancel', 'r': 'reset'}
+      axes = {'gb': ['w', 's'], 'steer': ['a', 'd']}
+      self.axes = {key: ax for ax in axes for key in axes[ax]}
     else:
-      self.buttons = {'BTN_TRIGGER': 'cancel', 'BTN_THUMB': 'engaged_toggle', 'BTN_THUMB2': 'reset'}
+      self.buttons = {'BTN_TRIGGER': 'cancel'}
       self.axes = {'ABS_Y': 'gb', 'ABS_X': 'steer'}
-    self.handle_button('reset')
-
-  def handle_button(self, btn_name):
-    if btn_name == 'reset':
-      self.axes_values = {ax: 0. for ax in self.axes.values()}
-      self.btn_states = {btn: False for btn in self.buttons.values() if btn != 'reset'}
-    else:
-      self.btn_states[btn_name] = not self.btn_states[btn_name]
+    self.axes_values = {ax: 0. for ax in self.axes.values()}
 
   def update(self):
     # Get key or joystick event
+    self.cancel = False
```

Comment:
If we don't care about sending a cancel request (can do it via vehicle CC stalk) we can get rid of all button code